recipe/meta.yaml

```diff
@@ -1,5 +1,5 @@
 {% set name = "bazel-toolchain" %}
-{% set version = "0.1.5" %}
+{% set version = "0.2.5" %}
```

Comment:
Any reason not to reset the patch version when bumping minor?

Reply:
> Do you think bumping the minor here is warranted anyway?

No idea; I had assumed it was intentional. Something for @conda-forge/bazel-toolchain to decide...